src/TeamSpeak3QueryApi/TeamSpeak3QueryApi.csproj

```diff
@@ -103,9 +104,6 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
-  <ItemGroup>
-    <Analyzer Include="..\packages\ConfigureAwaitChecker.Analyzer.1.0.0-beta4\analyzers\dotnet\cs\ConfigureAwaitChecker.dll" />
-  </ItemGroup>
```

Comment:
This analyzer is used to see if there could be some ConfigureAwaits put somewhere. We need this as part of the build. Could you revert this removal?